src/csharp/Grpc.IntegrationTesting/Payloads.cs

```diff
@@ -0,0 +1,580 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
As this file is generated code, is it supposed to be included in the repo? Does it need a review?